frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx

```diff
@@ -1,23 +1,30 @@
 import * as React from 'react';
 import { TFunction } from 'i18next';
 import { Shortcut, ShortcutTable } from '@console/shared';
+import { TopologyViewType } from '../../topology-types';
 
 export type Options = {
   supportedFileTypes: string[];
+  isEmptyModel: boolean;
+  viewType: TopologyViewType;
 };
 export const getTopologyShortcuts = (t: TFunction, options: Options): React.ReactElement => (
   <ShortcutTable>
```

Comment:
I'm a bit confused with this being a util. It could simply be a component and to me it looks more like a component.
I could be missing something here. Is there a reason we chose to make this a util rather than a component?

Reply:
although, it isn't really related to the problem that this bug is about.

Reply:
I doubt if there's any specific reason why this is a util and not a component

Reply:
I think it would be a good idea to convert this to a component. But since its not related to the changes in this PR, I would let this go in without it as well.